test/smoke/testApps/TelemetryProcessors/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/SpringBootAutoTest.java

```diff
@@ -44,8 +44,10 @@ public void doMostBasicTest() throws Exception {
     assertTrue(telemetry.rd.getSuccess());
     // Log processor test
     List<MessageData> logs = mockedIngestion.getMessageDataInRequest();
-    MessageData md1 = logs.get(0);
-    assertEquals("smoketestappcontroller::INFO", md1.getMessage());
+    if (!logs.isEmpty()) {
+      MessageData md1 = logs.get(0);
+      assertEquals("testValue1::testValue2", md1.getMessage());
+    }
```

Comment:
do we need the conditional?

```suggestion
    MessageData md1 = logs.get(0);
    assertEquals("testValue1::testValue2", md1.getMessage());
```